### ISSUE TITLE
Scale issues?

### ISSUE THREAD

**Issue by woodwardtw** [vcualtlab/bensonstats#4]
Seems like it never finishes loading. [David's commenting view](http://rampages.us/grandma-socy/conversation-view/) is experiencing this at the moment with 422 comments.

**Comment by luetkemj** [comment]
Yep - 502's at 254 posts. Wp-engine is killing the script after about a second and a half.
